.github/workflows/ci.yml

```diff
@@ -0,0 +1,116 @@
+# Used as inspiration: https://github.com/mvdan/github-actions-golang
+
+name: Cross-Platform
+
+on:
+  push:
+    branches: 
+      - v2
+  pull_request:
+    branches: 
+      - v2
+
+jobs:
+  test:
+    strategy:
+      # Default is true, cancels jobs for other platforms if one fails
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
+        go-version: [ 1.14.x ]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v1
+      with:
+        go-version: ${{ matrix.go-version }}
+
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    # - name: Install test and coverage analysis tools
+    #   run: |
+    #     go get github.com/axw/gocov/gocov
+    #     go get github.com/AlekSi/gocov-xml
+    #     go get -u github.com/jstemmer/go-junit-report
+    #     echo "::add-path::$(go env GOPATH)/bin"
```

Comment:
For the commented lines, can we add additional comment explaining why it's commented? #meta

(For all the commented sections, not just this one)

Reply:
Done 👍

Reply:
LOVE 'em, thanks!